### PR TITLE
Update annotations for tasks

### PR DESCRIPTION
**Pull request by henilp105**
This PR updates metadata for various tasks as they were previously partially filled. 
## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

Thanks and regards,
Henil

### CODE REVIEW

**Comment by henilp105** [comment]
@KennethEnevoldsen @isaac-chung can you please review this if possible ! Thanks.

**Review by KennethEnevoldsen — APPROVED** [review]

File: docs/mmteb/points/936.jsonl [hunk 1]
```diff
@@ -0,0 +1 @@
+{"GitHub": "henilp105", "Dataset annotations": 9}
```
Comment (KennethEnevoldsen) [inline docs/mmteb/points/936.jsonl:1]:
```suggestion
{"GitHub": "henilp105", "Dataset annotations": 9}
{"GitHub": "KennethEnevoldsen", "Review PR": 2}
```

**Comment by KennethEnevoldsen** [comment]
This looks great! I have set up auto-merge

**Comment by henilp105** [comment]
Thanks @KennethEnevoldsen , can you please merge it manually as due to linting the auto merge is disabled.

**Pull request merged**